config/settings.json

```diff
@@ -32,6 +34,14 @@
         "enabled": {
             "shipping": "shipping-basic",
             "discount": "discount-voucher"
+        },
+        "loaded": {
+            "shipping": {},
+            "discount": {}
         }
-    }
-}
+    },
+    "secretCookie": "24565e652c5baf4a225a6381615b3896abb53bff",
+    "secretSession": "3bfcadcc2de9ac9bc3e8ddbe449585cf7fdb744f",
```

Comment:
Why is this used for?

Reply:
This PR is closed but those values are used for `cookie-parser` module here: https://github.com/expressjs/cookie-parser#cookieparsersecret-options